### DISCUSSION TITLE
Help in 'slice' used in forward function

### DISCUSSION THREAD

**Discussion by Sindhu-Hegde** [snakers4/silero-models#62] [Q&A]
You have used the following lines in the forward function of model:
```
  _4 = torch.slice(x0, 0, 0, 9223372036854775807, 1)
  _5 = torch.slice(_4, 1, 0, 9223372036854775807, 1)
  _6 = torch.slice(_5, 2, 0, 9223372036854775807, 1)
  x_real = torch.select(_6, 3, 0)
```
Could you briefly explain what is the purpose of `torch.slice` here?

**Comment by snakers4** [comment]
this is a snippet from a torchscript model
it is not meant to be human readable
but I would suppose that this is feature extraction, which you do not really need to change for your purpose
